File: MyLanguageUtils/src/main/java/ru/herooo/mylanguageutils/file/mimetype/ImageMimeTypes.java
package ru.herooo.mylanguageutils.file.mimetype;

public enum ImageMimeTypes {
    ALL(new MimeType("image", "*")),
    PNG(new MimeType("image", "png")),
    JPG(new MimeType("image", "jpg")),
    JPEG(new MimeType("image", "jpeg")),
    GIF(new MimeType("image", "gif"))

    ;

    public final MimeType MIME_TYPE;

    ImageMimeTypes(MimeType mimeType) {
        this.MIME_TYPE = mimeType;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/html/input/file/mime_type_with_size/mime_type/types/image_mime_types.js
import {
    MIMEType
} from "../mime_type.js";

export class ImageMIMETypes {
    #TYPE = "image";

    ALL = new MIMEType(`${this.#TYPE}/*`, "*");
    PNG = new MIMEType(`${this.#TYPE}/png`, ".PNG");
    JPG = new MIMEType(`${this.#TYPE}/jpg`, ".JPG");
    JPEG = new MIMEType(`${this.#TYPE}/jpeg`,".JPEG");
    GIF = new MIMEType(`${this.#TYPE}/gif`, ".GIF");
}